backend/src/Controller/StoreOwnerBranch/StoreOwnerBranchController.php

```diff
@@ -179,6 +181,78 @@ public function createMultipleBranches(Request $request): JsonResponse
         return $this->response($result, self::CREATE);
     }
 
+    /**
```

Comment:
all APIs related to admin use, move it to src/controller/admin/store/newcontroller

so you can add a special route name space for it.